dom-manipulation/main.js

```diff
@@ -0,0 +1,20 @@
+let clickCounter = 0;
+const $hotBtn = document.querySelector('.hot-button');
+const $clkCount = document.querySelector('.click-count');
+$hotBtn.addEventListener('click', () => {
+  clickCounter++
+  document.querySelector(".click-count").textContent = clickCounter;
+  if(clickCounter < 4) {
+  document.querySelector(".hot-button").className = "hot-button cold";
```

Comment:
You already have a variable that references your `hot-button` element, so just use that variable name instead of querying for the DOM again